src/lib/storage/chunk.hpp

```diff
@@ -151,6 +152,19 @@ class Chunk : private Noncopyable {
    */
   size_t estimate_memory_usage() const;
 
+  void increase_invalid_row_count(uint64_t count) const;
+
+  uint64_t invalid_row_count() const { return _invalid_row_count; }
+
+  /**
+   * The MvccDeletePlugin implements a logical chunk deletion (full chunk invalidation) as a transaction.
+   * In case a clean-up transaction has been performed successfully its commit-id will be returned via
+   * this function. Otherwise, MvccData::MAX_COMMIT_ID is returned.
+   */
+  CommitID get_cleanup_commit_id() const { return _cleanup_commit_id; }
```

Comment:
Is the concept of cleanup commit IDs documented somewhere? If not, this would be a good place to do so.